README.md

```diff
@@ -1,41 +1,73 @@
+
+## Template Versions
+
+#### Accessing Different CLI Versions of Templates
+This template is compatible with version(s) of the Slack CLI.
+
+To view a template that is compatible with a specific version of the CLI, view the list below:
+
+- **(current)**[deno-starter-template for CLI v1.0.0](https://github.com/slack-samples/deno-starter-template/tree/main)
+
+## Installation
+
+#### Prerequisites
 
 To use this template, you will need to have installed and configured the Slack CLI. 
 Do this by following our [Quickstart Guide](https://api.slack.com/future/quickstart).
 
-# Making an app
+You will also need a `slack.json` file - you can learn more about this file in the `Project Structure` section.
 
-Make a Run On Slack app with this with repo by **creating an app from this template**, 
-**configuring your app**, then **writing functions**. 
+#### Get Started
 
-Once you're done, you can Run your app with the CLI's local development server
-or Deploy your app to production.
+Make a Run On Slack app with this with repo by **creating an app from this template**, **configuring your app**, then **writing functions**. 
 
-## 1. Creating an app from this template
+Once you're done, you can Run your app with the CLI's local development server or Deploy your app to production.
 
-Clone this repo with the Slack CLI:
+### Setup Your Project
 
 ```zsh
-$ slack create my-app -t slack-samples/deno-starter-template
+# Clone this project onto your machine
+slack create my-app -t slack-samples/deno-starter-template
+
+# Change into this project directory
+cd my-app
+
+# Run app locally
+slack run
+
+# Deploy app
+slack deploy
 ```
 
-## 2. Configuring your app
+#### Running your app locally
 
-The first thing we'll do is configure our app's manifest. This will let us 
-configure things like our app's name and the [scopes](https://api.slack.com/scopes)
-it requires.
+While building your app, you can see your changes propagated to your workspace in real-time with `slack run`.
 
-Enter your project's root directory:
+Executing `slack run` starts a local development server, syncing changes to your workspace's development version of your app. (You'll know it's the development version because the name has the string `(dev)` appended).
+
+Your local development server is ready to go when you see the following:
 
 ```zsh
-$ cd my-app
+Connected, awaiting events
+
 ```
 
-Next, open the manifest file, `manifest.ts`. 
+When you want to turn off the local development server, use `Ctrl+c` in the command prompt.
+
+#### Deploying your app to Slack
+
+When you're done developing your app, you can deploy it directly to Slack with `slack deploy`.
+
+#### Configuring your app
+
+The first thing we'll do is configure our app's manifest. This will let us configure things like our app's name and the [scopes](https://api.slack.com/scopes) it requires.
```

Comment:
If configuring is the first step, should it not come before instructions on how to run the app? Also, shouldn't deploy go very last?